docker/src/main/java/brooklyn/entity/container/docker/DockerContainerImpl.java

```diff
@@ -614,13 +614,15 @@ public DockerContainerLocation createLocation(Map flags) {
             }
 
             // Create our wrapper location around the container
-            LocationSpec<DockerContainerLocation> spec = LocationSpec.create(DockerContainerLocation.class)
+            LocationSpec<DockerContainerLocation> spec = (LocationSpec<DockerContainerLocation>) LocationSpec.create(DockerContainerLocation.class)
```

Comment:
This compiles, might be neater than casting:
```java
LocationSpec<DockerContainerLocation> spec = LocationSpec.create(DockerContainerLocation.class);
            spec.parent(host)
            .configure(flags)
...
```

but still not sure what has changed to make this required